berserk/clients/studies.py

```diff
@@ -24,3 +24,10 @@ def export(self, study_id: str) -> Iterator[str]:
         """
         path = f"/api/study/{study_id}.pgn"
         yield from self._r.get(path, fmt=PGN, stream=True)
+
+    def get_study_by_username(
```

Comment:
`client.studies.get_study_by_username` is probably too verbose, I'd rather go with just `by_user`